sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryOptions.java

```diff
@@ -122,6 +142,9 @@ public interface BigQueryOptions
 
   void setStorageWriteMaxInflightBytes(Long value);
 
+  @Description(
+      "Enables multiplexing mode, where multiple tables can share the same connection. Only available when writing with STORAGE_API_AT_LEAST_ONCE"
+          + " mode. For more information, see https://cloud.google.com/bigquery/docs/write-api-best-practices#connection_pool_management")
```

Comment:
Could we add that
1) this is recommended when writing to >20 tables - I think this is true based on the linked doc, if not maybe we can add other guidance on when to do this
2) Can we add something like "when using multiplexing mode, consider limiting `MaxConnectionPoolConnections` as well"?

Reply:
Done